client/src/components/SignCampaign.js

```diff
@@ -70,73 +98,100 @@ class SignCampaign extends Component {
+                />
+                {this.createCheckboxes()}
+                <div className="sign-campaign-actions">
+                  <Button
+                    className="remove-default sign-petition-button"
+                    value="submit"
+                    type="submit"
+                    block
+                  >
+                    Sign the petition
+                  </Button>
+                </div>
+              </FormGroup>
+            </form>
+          )}
+      </div>
+    );
+
+    // user is signed in && has signed campaign
+    const renderRemoveSignaturePrompt = (
+      <div>
+        {signedIn &&
+          userHasSignedThisCampaign && (
+            <div>
+              <ToolList />
               <div className="text-center">
-                <Button className="logout-button-signature" onClick={this.handleSignOut} block>
-                  Sign Out
-                  <i className="fa fa-sign-out" />
-                </Button>
-              </div>
-            </Col>
-          </Row>
-        </div>
-      );
-    }
-    // User is signed in but hasn't signed campaign
-    return (
-      <Row>
-        <Col md={12}>
-          <form onSubmit={this.handleFormSubmit}>
-            <FormGroup controlId="signingBecause">
-              <ControlLabel id="control-label">
-                <h4>I'm signing because...</h4>
-              </ControlLabel>
-              <FormControl
-                className="form-resize-vertical"
-                componentClass="textarea"
-                placeholder="Optional"
-              />
-              {this.createCheckboxes()}
-              <div className="sign-campaign-actions">
                 <Button
-                  className="remove-default sign-petition-button"
-                  value="submit"
-                  type="submit"
+                  className="logout-button-signature"
+                  onClick={this.handleRemoveSignature}
                   block
                 >
-                  Sign the petition
+                  <i className="fa fa-times-circle" />
+                  Unsign This Campaign
                 </Button>
               </div>
-              <h5 className="content remove-margin text-center">OR</h5>
-            </FormGroup>
-            <Button className="logout-button-signature" onClick={this.handleSignOut} block>
-              Sign Out
-              <i className="fa fa-sign-out" />
+            </div>
+          )}
+      </div>
+    );
+
+    // user is signed in && is currently on a page different from their signed campaign page
+    const renderUserHasSignedOtherCampaignPrompt = (
+      <div className="user-has-signed-other-campaign-alert">
+        {userHasSignedOtherCampaign && (
+          <Alert bsStyle="warning" onDismiss={this.handleDismiss}>
+            <h4>Oh snap!</h4>
+            <p>
+              Looks like you have already signed a campaign, you little overachiever you. You can
```

Comment:
Perhaps swap "You can only sign one campaign at a time" with "Oh Snap". It will look better in bold and it's the most important part of the alert?

Reply:
Ok